code/datums/uplink_item.dm

```diff
@@ -714,7 +714,7 @@ var/list/uplink_items = list()
 		buyable_items += temp_uplink_list[category]
 	var/list/bought_items = list()
 	U.uses -= cost
-	U.used_TC = 20
+	U.used_TC = 40
 	var/remaining_TC = 50

```

Comment:
this needs to be 100 now.

Previously you paid 20 and got 50 TC---meaning effectively x2.5 TC; if you're really doubling TC, then paying 40 to get an extra free 10 TC will almost never be worth it.